### PR TITLE
feat: make appium timeout and time between snaps declarable from cli

### PR DESCRIPTION
**Pull request by rjanjua**

### CODE REVIEW

File: bin/run.js [hunk 1]
```diff
@@ -5,15 +5,19 @@ const resolve = require('path').resolve
...
+  .option("-T, --timeout [timeout]", "the timeout applied to appium before it closes the app", parseInt)
   .parse(process.argv);
 
 
-if (!program.config)
-  throw new Error(
-    "Please specify the dextrose config: --config [directory/config.js]"
-  );
+if (!program.config){
+  console.log("Please specify the dextrose config: --config [directory/config.js]");
```
Comment (kaliabadi) [inline bin/run.js]: Does this need to be a warn or error or something?

Comment (rjanjua) [reply bin/run.js]: it's a message about why we can't execute the command. Maybe we should use a proper logger

Comment (kaliabadi) [reply bin/run.js]: Ah yeah I see, ok well it's fine for now but we should use a proper logger. I will raise a separate issue.

Comment (L0wry) [reply bin/run.js]: we have got a proper logger in dextrose. It's called `logger` under `src`

File: bin/run.js [hunk 2]
```diff
@@ -5,15 +5,19 @@ const resolve = require('path').resolve
 
 program
   .version("0.0.1")
-  .option("-c, --config [dextrose-config]", "please specify the dextrose config")
+  .option("-c, --config [dextrose-config]", "A file that exports the dextrose config for your app")
+  .option("-t, --snapshotWait [snapshot-wait]", "the amount of timeout to wait between loading a component and taking the snap", parseInt)
```
Comment (L0wry) [inline bin/run.js]: `the amount of timeout to wait`

**Pull request merged**